01_P/P_2_1_2_02/sketch.js

```diff
@@ -42,29 +42,26 @@ var moduleAlphaForeground = 100;
 var moduleRadiusBackground = 30;
 var moduleRadiusForeground = 15;
 
-var backColor;
-
+var backgroundColor;
 
 var tileCount = 20;
 var actRandomSeed = 0;
 
 function setup(){
   createCanvas(600, 600);
-
   colorMode(HSB, 360, 100, 100, 100);
+  noStroke();
 
   moduleColorBackground = color(0, 0, 0, moduleAlphaBackground);
   moduleColorForeground = color(0, 0, 100, moduleAlphaForeground);
 
-  backColor = color(0, 0, 100);
+  backgroundColor = color(0, 0, 100);
 }
 
 function draw() {
   translate(width/tileCount/2, height/tileCount/2);
 
-  background(backColor);
-  smooth();
```

Comment:
Removed `smooth()` since this is already enabled by default in every p5 sketch (as well as Processing 2 and 3). Moved `noStroke` into `setup` because it only has to be set once.